### PR TITLE
 Improving storage

### PR DESCRIPTION
**Pull request by taiebchaabini**
Update to DBStorage and FileStorage

### CODE REVIEW

File: models/engine/file_storage.py [hunk 1]
```diff
@@ -24,6 +24,28 @@ class FileStorage:
...
+            return None
+        return obj
+
+    def count(self, cls=None):
+        """
+        Count the number of objects in storage:
+        @cls: class name
+        Return:
+        """
+        objs = self.all(cls)
```
Comment (jenntang1) [inline models/engine/file_storage.py:46]: Incorporate this into the return statement for a super concise method

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -40,6 +40,27 @@ def __init__(self):
...
+            return None
+        return obj
+
+    def count(self, cls=None):
+        """
+        Count the number of objects in storage:
+        @cls: class name
+        Return:
+        """
+        objs = self.all(cls)
```
Comment (jenntang1) [inline models/engine/db_storage.py:61]: Incorporate this into the return statement for a super concise method

**Review by jenntang1 — COMMENTED** [review]
Great job!

File: models/engine/db_storage.py (hunk 2 above)
Comment (taiebchaabini) [reply models/engine/db_storage.py:61]: Thank you !

File: models/engine/file_storage.py (hunk 1 above)
Comment (taiebchaabini) [reply models/engine/file_storage.py:46]: Thank you !

**Pull request merged**